File: tools/dbench/answers/go_haiku/s3/f1_users.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	file, _ := os.Open("users.txt")
	defer file.Close()

	scanner := bufio.NewScanner(file)
	validCount := 0
	ageSum := 0

	for scanner.Scan() {
		line := scanner.Text()
		fields := strings.Fields(line)
		if len(fields) != 3 {
			continue
		}
		age, err := strconv.Atoi(fields[2])
		if err != nil {
			continue
		}
		validCount++
		ageSum += age
	}

	fmt.Println(validCount)
	if validCount > 0 {
		fmt.Println(ageSum / validCount)
	}
}

File: tools/dbench/answers/python_sonnet/s2/f1_users.py
ages = []
with open("users.txt") as f:
    for line in f:
        p = line.split()
        if len(p) == 3:
            try:
                ages.append(int(p[2]))
            except ValueError:
                pass
print(len(ages))
print(sum(ages) // len(ages))
